src/update/utils.py

```diff
@@ -0,0 +1,42 @@
+"""
+A few project level Enums and utils
+"""
+import argparse
+from enum import Enum
+
+from duneapi.types import QueryParameter
+
+from src.logger import set_log
+
+log = set_log(__name__)
```

Comment:
Random question for discussion: Can't we set the logger in one place and then just always import it from here?

Reply:
unfortunately I do not believe so, because we have to pass `__name__` into it. I would love it if you had a way around this.

Reply:
Unfortunately I'm currently preparing for the monthly reporting meeting and don't have a lot of time to look into this. I've created an issue for it #126